lambda/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM public.ecr.aws/lambda/python:3.8
 COPY requirements.txt ${LAMBDA_TASK_ROOT}
-RUN pip3 install --no-cache-dir -r requirement.txt
+RUN pip3 install --no-cache-dir -r requirements.txt
```

Comment:
This is a typo. Line 2 of the Dockerfile clearly shows `requirements.txt` as the file name. As such, the Dockerfile would have failed on line 3 with a "file not found" error.